internal/npm_install/generate_build_file.js

```diff
@@ -768,7 +768,7 @@ node_module_library(
     # ${pkg._dir} package contents (and contents of nested node_modules)
     srcs = [
         ":${pkg._name}__files",
-    ],${depsStarlark}
+    ],${depsStarlark}${scriptStarlark}
```

Comment:
Moving `scripts` to __pkg allows removing

```
    for src in ctx.attr.srcs:
        if NodeModuleSources in src:
            scripts = depset(transitive = [scripts, src[NodeModuleSources].scripts])
```

from node_module_library.bzl. __pkg is the standard dep for users and if a user is using __files then they are being even more fine grained with their deps and probably don't want scripts automatically ending up in ts_devserver & ts_web_test_suite